apps/redi-connect/src/components/organisms/modules/MSessions.tsx

```diff
@@ -86,6 +87,7 @@ const MSessions = ({
       await mentoringSessionsCreateStart(mentoringSession)
       setSubmitResult('success')
       setShowAddSession(false)
+      window.location.reload()
```

Comment:
What's this good for?

Reply:
Ahh, I now get it after reviewing #350. This is really a work-around solution. The underlying problem should get fixed when the data layer gets replaced by end of July. However, let's leave in place this "band-aid" solution until then.

Reply:
It is but as you said: once the underlying problem is fixed this should be gone as well. So this is good to go for you?

Reply:
@marcuszierke hi there, sorry for the delay here. Crazy week with two launches.

Reply:
Yes; this is a good-to-go from me